src/sentry/models/actor.py

```diff
@@ -63,8 +65,8 @@ def actor_type_to_string(type: int) -> Optional[str]:
     return None
 
 
-@region_silo_only_model
-class Actor(Model):
+@control_and_region_silo_model
```

Comment:
Ah hah, so the intention is to replicate `Actor` in both control and region silos?

If that's the case, I'd still ponder if it makes sense not to use the same model, but actually to have 2 duplicate models, a "region" actor and a "control" actor.  There are advantages to doing this:

1.  Makes it explicitly clear to the programmer that writing an actor does not magically make it synchronously appear in other silos -- they are separate models and most be synchronized by some other mechanism (in fact, I'm working on that right now)
2.  Makes our other various safeguards work better, which are written assuming that each individual model class is only either region or control silo.

I think if you take the approach of duplicating the actor model into two, one for each region, things may work out better.

Reply:
The idea was to have the Actor model in both control and region silos. It seems we may not need to replicate Actor data across silos because Actor is an internal concept.

It is the User and Team models that are external facing, i.e. we get API requests referencing users and teams and not actors.

Reply:
> If that's the case, I'd still ponder if it makes sense not to use the same model

Yes I was wondering if it is feasible to have a model in both control and region silos without being replicated.
Makes sense not to have that to keep the design simple.